frontend/src/features/mapSideBar/containers/MapSideBarContainer.tsx

```diff
@@ -117,12 +124,16 @@ const MapSideBarContainer: React.FunctionComponent<IMapSideBarContainerProps> =
   ) => {
     fetchParcelsDetail(pidOrPin)(dispatch).then(resp => {
       const matchingParcel: any = resp?.data?.length ? _.first(resp?.data) : undefined;
-      if (!!nameSpace && !!formikRef?.current?.values && !!matchingParcel?.id) {
+      if (!!formikRef?.current?.values && !!matchingParcel?.id) {
         const { resetForm, values } = formikRef.current;
-        matchingParcel.searchPid = getIn(values, `${nameSpace}.searchPid`);
-        matchingParcel.searchPin = getIn(values, `${nameSpace}.searchPin`);
-        matchingParcel.searchAddress = getIn(values, `${nameSpace}.searchAddress`);
-        resetForm({ values: setIn(values, nameSpace, matchingParcel) });
+        matchingParcel.searchPid = getIn(values, withNameSpace('searchPid', nameSpace));
+        matchingParcel.searchPin = getIn(values, withNameSpace('searchPin', nameSpace));
+        matchingParcel.searchAddress = getIn(values, withNameSpace('seachAddress', nameSpace));
+        matchingParcel.financials = getMergedFinancials([
```

Comment:
if I get time I'll update the evaluation form so that it doesn't need this financials array anymore and uses evaluations and fiscals directly.